analyticConfigurations/computed_metric-aws.tag.json

```diff
@@ -0,0 +1,23 @@
+{
+  "analyticConfiguration" : {
+    "tenantId" : null,
+    "type" : "COMPUTED_METRIC",
+    "id" : null,
+    "packageId" : null,
+    "name" : "AWS Target Group",
+    "scope" : {
+      "elementName" : null,
+      "elementType" : "Target Group",
+      "elementTags" : null,
+      "metricMatches" : null
+    },
+    "metrics" : [ {
+        "match": "netuitive.aws.applicationelb.httpsuccesspercent",
+        "properties": {
+          "cml": "100 * ${aws.applicationelb.httpcode_target_2xx_count}.actual / (${aws.applicationelb.requestcount}.actual - ${aws.applicationelb.httpcode_target_3xx_count}.actual - ${aws.applicationelb.httpcode_target_4xx_count}.actual)",
```

Comment:
As per the ongoing discussion, @rclagett and I would rather this metric reflect total HTTP success rate (2xx_count/requestcount) and, if desired, another metric should be created for this calculation which leaves out 3xx and 4xx.

Reply:
Can you explain further?  Would you still call it "httpsuccesspercent"?  The "httpsuccesspercent" would be artificially skewed by successful requests to the service such as redirects, unauthorized responses etc.  why include these in the calculation?

Reply:
I'll grant that 3xx are likely successful, but are 4xx successful requests? Even if so, shouldn't you add 3xx and 4xx to the 2xx and divide that number by 
the total, rather than subtracting 3xx and 4xx from the total in the denominator?

Reply:
That may be more accurate calculated that way - @TheConnMan ?  
I can see the grey area with client errors, it could very well be a server side issue that causes false positives like "not found" or "forbidden".

Reply:
I'm fine with either just 2xxs or 200-499s. I'd opt for 2xx + 3xx + 4xx and if there are too many 3xx or 4xx then it encourages us to investigate and prevent those from coming in.

Reply:
ok let's do it with 2xx + 3xx + 4xx and, please add another metric with the calculation which leaves out 3xx and 4xx.